src/containers/test/utils.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC, PropsWithChildren } from 'react'
+import { HelmetProvider } from 'react-helmet-async'
+import { I18nextProvider } from 'react-i18next'
+import { QueryClientProvider } from 'react-query'
+import { MemoryRouter } from 'react-router'
+import type i18n from '../../i18n/testConfig'
+import { testQueryClient } from './QueryClient'
+
+export function flushPromises() {
+  return new Promise((resolve) => setImmediate(resolve))
+}
+
+export const QuickHarness: FC<
```

Comment:
I don’t like the name. I’d go with something about app wrapper or app provider

Reply:
This is a testing harness.

Reply:
TestingProvider? It’s just a nit